src/System.Memory/src/System.Memory.csproj

```diff
@@ -33,6 +33,9 @@
     <Compile Include="System\Buffers\Binary\Writer.cs" />
     <Compile Include="System\Buffers\Binary\WriterBigEndian.cs" />
     <Compile Include="System\Buffers\Binary\WriterLittleEndian.cs" />
+    <Compile Include="System\Buffers\MemoryPool.cs" />
+    <Compile Include="System\Buffers\ArrayMemoryPool.cs" />
+    <Compile Include="System\Buffers\ArrayMemoryPool.ArrayMemoryPoolBuffer.cs" />
```

Comment:
nit: fix sort order. put it above 
`<Compile Include="System\Buffers\OperationStatus.cs" />`

Reply:
The double-dot notation is used generally when partial classes are used. This is just another instance of that.